src/pages/tBTC/Bridge/Minting/Timeline.tsx

```diff
@@ -0,0 +1,196 @@
+import { FC, ReactNode, useMemo } from "react"
+import {
+  Box,
+  BodyLg,
+  BoxProps,
+  Accordion as TimelineContainer,
+  Badge,
+  BadgeProps,
+} from "@threshold-network/components"
+import TimelineItem, { TimelineItemProps } from "../components/TimelineItem"
+import { useTbtcState } from "../../../../hooks/useTbtcState"
+import { DepositDetailsStep, MintingStep } from "../../../../types/tbtc"
+import { useWeb3React } from "@web3-react/core"
+import { useLocation } from "react-router"
+
+const StyledBadge: FC<Omit<BadgeProps, "variant" | "mt" | "marginTop">> = (
+  props
+) => {
+  const { account } = useWeb3React()
+  return account ? null : (
+    <Badge
+      variant="subtle"
+      width="fit-content"
+      display="block"
+      mt={4}
+      {...props}
+    />
+  )
+}
+
+type TimelineItemType = {
+  id: DepositDetailsStep | MintingStep
+  label: string
+  description: ReactNode
+}
+
+const mintingItems: TimelineItemType[] = [
```

Comment:
For static data such as this, we should use uppercase snake case as we use it across the app:

```suggestion
const MINTING_ITEMS: TimelineItemType[] = [
```

It will clearly indicate that data in this variable is static.